Update pen-haptics.md

ryan-forstie opened the pull request.

ryan-forstie committed:
Update pen-haptics.md

PRMerger17 commented:
@ryan-forstie : Thanks for your contribution! The author(s) have been notified to review your proposed change.

Karl-Bridge-Microsoft committed:
Update pen-haptics.md

Revised: Trimmed word count and moved adjacent to SDK waveform name (instead of disconnected footnote)

PRMerger10 commented:
@Karl-Bridge-Microsoft : Thanks for your contribution! The author(s) have been notified to review your proposed change.
